### PR TITLE
feat: backdrop-filter

### PR DESCRIPTION
**Pull request by ThatOneCalculator**
Adds backdrop filter for blurs/etc

image

### CODE REVIEW

**Comment by kolirt** [comment]
@ThatOneCalculator 

Hi. Thank you for your PR
Could you run ```yarn format``` and update PR?

File: bun.lockb
Comment (kolirt) [inline bun.lockb:1]: remove this file

File: examples/vite/src/main.ts
Comment (kolirt) [inline examples/vite/src/main.ts:1]: run ```yarn format```

File: examples/vue-cli-service/src/main.js
Comment (kolirt) [inline examples/vue-cli-service/src/main.js:1]: run ```yarn format```

**Comment by kolirt** [comment]
@ThatOneCalculator 

Please upgrade to v1.0.1

**Comment by ThatOneCalculator** [comment]
Generally bun.lockb shouldn't be gitignored

**Comment by kolirt** [comment]
> Generally bun.lockb shouldn't be gitignored

Yes, I know, but it doesn't use for deploying and development by me. So I deleted it
In general, you are right

**Pull request merged**